### ISSUE TITLE
[Bug] Source Nhattruyen can't load all the way to the last page

### ISSUE THREAD

**Issue by vudoanbinhminh** [tachiyomiorg/tachiyomi-extensions#5543]
## Device information
* Tachiyomi version: 0.10.7
* Android version: 10
* Device: Realme 3 Pro 

## Steps to reproduce
1. Read Kimi Wa 008 ( http://nhattruyen.com/truyen-tranh/cau-la-008/chap-48/679466 )
2. tachiyomi will only load the first few images, then blocked by nettruyen logo

### Expected behavior
load full manga 

### Actual behavior
tachiyomi will only load the first few images, then blocked by nettruyen logo

## Other details
I think the problem is that these manga are nettruyen's own source (other manga they get from outside are still read normally) + it has logo nettruyen 

Plus :  The same thing happened with mangasum source

![Screenshot_2021-01-23-11-34-02-99_5901a79de973a50c74cff4cbb6149856](https://user-images.githubusercontent.com/33306530/105622950-47314680-5e48-11eb-9442-b0e8f77ef75e.jpg)

**Comment by Riztard** [comment]
it load for me, what is your extension version? or try [troubleshoot](https://tachiyomi.org/help/guides/troubleshooting-problems/)

**Comment by scb261** [comment]
I was able to reproduce it before, but now it works fine. I didn't look at html, but it looks like they changed loading back to normal - loading icon and their logo were overlapping in webview/browser before.

**Comment by scb261** [comment]
Doesn't happen to me anymore so closing it. Open new issue if it still happens after removing downloaded chapters with the issue and clearing chapter cache in advanced settings
